drivers/gpu/drm/panel/panel-samsung-s6e3fa0-full.c

```diff
@@ -0,0 +1,764 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
this file should not be named _full either

Reply:
Removed _full